lib/commands/get.js

```diff
@@ -45,21 +45,19 @@ module.exports = {
     let TabSelector = require('../tasks/select-tab.js');
     let tabSelector = new TabSelector({ project: this.project, tabName: options.tab });
 
-    return tabSelector.run().then(tab => this._rows(tab));
+    return tabSelector.run().then(tabs => {
+      return Promise.all(tabs.map(this._rows.bind(this))).then(rowsByTab => {
+        let mergedRows = [].concat.apply([], rowsByTab);
```

Comment:
not sure i understand what you concatenating here.

the then happens once and you doing a concat with []